### ISSUE TITLE
export format for issues with journal

### ISSUE THREAD

**Issue by johnp2686** [two-pack/redmine_xls_export#44]
Hello ,

I get the journals entries for i asked but i get in issue journals as separate file ,i need all issue in single xls sheet how export single sheet all issue i will send u code and screen shot

redmine_xls_export/lib/xls_export.rb

def journal_details_to_xls(issue)
issue_updates = issue.journals.all(:include => [:user, :details], :order => "#{Journal.table_name}.created_on ASC")
return nil if issue_updates.size == 0

```
Spreadsheet.client_encoding = 'UTF-8'
book = Spreadsheet::Workbook.new
sheet1 = book.create_worksheet(:name => "%05i - Journal" % [issue.id])

columns_width = []
sheet1.row(0).replace []
['#',l(:field_issue),l(:label_tracker),l(:field_priority),l(:field_created_on),l(:field_updated_on),l(:field_user),l(:field_status_from),l(:field_status_to)].each do |c|
  sheet1.row(0) << c
  columns_width << (get_value_width(c)*1.1)
end
sheet1.column(0).default_format = Spreadsheet::Format.new(:number_format => "0")

idx=0
issue_updates.each do |journal|
  row = sheet1.row(idx+1)
  row.replace []

  details=''
  journal.details.each do |detail|
    if detail.prop_key == 'status_id'
     [idx+1,issue.id,issue.tracker_id,issue.priority_id,issue.created_on,journal.created_on,journal.user.name,detail.old_value,detail.value].each_with_index do |e,e_idx|
        lf_pos = get_value_width(e)
        columns_width[e_idx] = lf_pos unless columns_width[e_idx] >= lf_pos
        row << e
      end
      idx=idx+1
    end
  end
```

end

```
update_sheet_formatting(sheet1,columns_width)

xls_stream = StringIO.new('')
book.write(xls_stream)
xls_stream.string
```

  end

i get only one issue but i want get all issue single sheet can u help regarding this issue it is need full very urgent
![journal_format](https://f.cloud.github.com/assets/951104/2039242/3c94ba62-899e-11e3-9feb-b6f1b3ba157a.png)

**Comment by two-pack** [comment]
Is your request that some columns are added in exported journal history?
The plugin already has exporting journal history which correspond issue page.
What is the purpose of this request format? How to use it? And Is it useful for all users?

**Comment by two-pack** [comment]
I implement the feature related this issue. See #47

**Comment by two-pack** [comment]
Fixed on #47.
